pilot/pkg/networking/core/v1alpha3/cluster_builder.go

```diff
@@ -610,63 +615,33 @@ func addUint32(left, right uint32) (uint32, bool) {
 func (cb *ClusterBuilder) buildInboundPassthroughClusters() []*cluster.Cluster {
 	// ipv4 and ipv6 feature detection. Envoy cannot ignore a config where the ip version is not supported
 	clusters := make([]*cluster.Cluster, 0, 2)
-	// There is an usage doc here:
-	// https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/core/v3/address.proto#config-core-v3-bindconfig
-	// to support the Dual Stack via Envoy bindconfig, and belows are related issue and PR in Envoy:
-	// https://github.com/envoyproxy/envoy/issues/9811
-	// https://github.com/envoyproxy/envoy/pull/22639
-	if cb.isDualStackCluster() {
-		inboundPassthroughCluster := cb.buildDefaultPassthroughCluster()
-		inboundPassthroughCluster.Name = util.InboundPassthroughCluster
-		inboundPassthroughCluster.Filters = nil
-		inboundPassthroughCluster.UpstreamBindConfig = &core.BindConfig{
+	if cb.supportsIPv4 {
```

Comment:
how come this is reverted? It seems desired...?

Reply:
Yes, @howardjohn, it may not be a proper time to merge them together, there are so many places still separately use `InboundPassthroughBindIpv4` and `InboundPassthroughBindIpv6`. In fact, we can use the previous submit to handle it, but it may be not a good solution.